src/uterm_input_uxkb.c

```diff
@@ -405,6 +483,10 @@ int uxkb_dev_process(struct uterm_input_dev *dev,
 	if (num_keysyms <= 0)
 		return -ENOKEY;
 
+	if (compose_status == XKB_COMPOSE_COMPOSING ||
+	    compose_status == XKB_COMPOSE_CANCELLED)
```

Comment:
Should this keysym be dropped or let through?

Reply:
Disclaimer: I don't use composing, so take what I said for this with a grain of salt.

I think it makes sense to drop like what you've done here. It's consistent with how X handles it.